ddp-workspace/projects/ddp-sdk/src/lib/components/internationalization/languageSelector.component.ts

```diff
@@ -267,13 +271,38 @@ export class LanguageSelectorComponent implements OnInit, OnDestroy {
 
     // Update the language in the profile to the current language
     private updateProfileLanguage(): Observable<any> {
-        const profileModifications: UserProfile = new UserProfile();
-        profileModifications.preferredLanguage = this.currentLanguage.languageCode;
-        return this.profileServiceAgent.updateProfile(profileModifications)
-            .pipe(
-                tap(() => this.afterProfileLanguageChange.emit()),
-                tap(() => this.language.notifyOfProfileLanguageUpdate())
-            );
+        if (this.config.updatePreferredLanguageForGovernedParticipants) {
```

Comment:
Based on the feedback in the ticket, is there an **easy** way to incorporate logic that is closer aligned to the response from Whitney?
If (this study allows governed participants) && (this study is translated to more than one language) then
    do this updated code
else
     do the original thing.